sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/Transforms.java

```diff
@@ -444,16 +446,15 @@ static BlobQuerySerialization toBlobQuerySerialization(FileQuerySerialization se
         } else if (ser instanceof FileQueryArrowSerialization) {
             FileQueryArrowSerialization arrSer = (FileQueryArrowSerialization) ser;
             return new BlobQueryArrowSerialization().setSchema(toBlobQueryArrowSchema(arrSer.getSchema()));
-            // TODO (gapra): uncomment when parquet is released
-        /*} else if (ser instanceof FileQueryParquetSerialization) {
-            return new BlobQueryParquetSerialization(); */
+        } else if (ser instanceof FileQueryParquetSerialization) {
+            return new BlobQueryParquetSerialization();
         } else {
             throw new IllegalArgumentException(
-                String.format("'serialization' must be one of %s, %s, or %s",
+                String.format("'serialization' must be one of %s, %s, %s, or %s",
```

Comment:
Do we want to make this error message into a constant instead of formatting every time it is thrown